### DISCUSSION TITLE
Spelling error on marketing site

### DISCUSSION THREAD

**Discussion by greyskymedia** [emqx/MQTTX#1112] [General]
Just a heads up, there's a spelling error on your marketing site home page. (Couldn't find any contact info to email it to)

<img width="803" alt="Screen Shot 2022-10-20 at 2 56 52 PM" src="https://user-images.githubusercontent.com/1276145/197066819-e8c45191-2e75-444f-8b32-faaef6fc38c5.png">

**Comment by robertmazzo** [comment]
the error is still there. I just noticed it a few min before reading this.

**Comment by Red-Asuka** [comment]
@greyskymedia Thank you for your feedback. I have fixed the spelling error on the website. You can contact us by filling out the form on the [contact us page](https://www.emqx.com/en/contact) or by using the email provided on the left side of the page. After you submit the form, our team will get in touch with you as soon as possible to provide the assistance you need.

**Reply by greyskymedia** [reply]
No assistance needed, just wanted you to know.
